packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -92,11 +92,28 @@ const formattedPlanTypes: Record<PlanType, string> = {
   [PlanType.OnPrem]: 'Ops Manager',
 } as const;
 
+const onKeyDown = (e: React.KeyboardEvent, setValue: Function) => {
```

Comment:
Heh, misunderstood.

This should literally be the setValue function from useState.

Cuz then in the body you can just do `setValue(currentValue => `${currentValue} `)`. That usage is consistent everywhere.

Reply:
🤦‍♀ got it